.env.example

```diff
@@ -159,3 +159,9 @@ TIMEZONE="Pacific Time (US & Canada)"
 
 # Number of failed jobs to keep in the database
 FAILED_JOBS_TO_KEEP=100
+
+# Maximum runtime of background jobs in minutes
+# DELAYED_JOB_MAX_RUNTIME=20
+
+# Amount of seconds for delayed_job to sleep before checking for new jobs
+# DELAYED_JOB_SLEEP_DELAY=10
```

Comment:
I do not have a strong opinion about having it comment our or enabled. The most important thing is that we also have the default in the code like you said.

I'll enable both DJ variables as well as the `SCHEDULER_FREQUENCY` to keep the `.env.example` consistent.